### PR TITLE
[1.6 Package Add] tenacity-git

### PR DESCRIPTION
**Pull request by rain1598**

### CODE REVIEW

File: packages/tenacity-git/tenacity-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,109 @@
+name="tenacity-git"
+version="r23.g098a0225e"
```
Comment (wizard-28) [inline packages/tenacity-git/tenacity-git.pacscript:2]: use `pkgver()` for auto-updates.

See [`awesome-git`](https://github.com/pacstall/pacstall-programs/blob/master/packages/awesome-git/awesome-git.pacscript) for an example.

File: packages/tenacity-git/tenacity-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+install() {
+          sudo make install DESTDIR="$STOWDIR/$name"
+}
+
+postinst(){
+          true
+}
+
+removescript(){
```
Comment (wizard-28) [inline packages/tenacity-git/tenacity-git.pacscript:33]: If I understand correctly `make` is installing this files in those directories despite specifying the `DESTDIR`, so you are manually deleting them right?

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/tenacity-git/tenacity-git.pacscript (hunk 2 above)
Comment (rain1598) [reply packages/tenacity-git/tenacity-git.pacscript:33]: yes

File: packages/tenacity-git/tenacity-git.pacscript [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+build_depends="build-essential cmake git python3-pip ffmpeg"
+depends="libgtk2.0-dev libasound2-dev libavformat-dev libjack-jackd2-dev uuid-dev"
+replace="audacity"
+description="FLOSS Audacity Fork. No telemetry, crash reports and other shenanigans like that!"
+optdepends="ffmpeg: additional import/export capabilities"
+removescript="yes"
+maintainer="echometer <guacamolek@yandex.com>"
+
+pkgver() {
+    git ls-remote "$(url)" master | cut -f1
```
Comment (wizard-28) [inline packages/tenacity-git/tenacity-git.pacscript:12]:
```suggestion
    git ls-remote "${url}" master | cut -f1 
```

You used the wrong braces.

File: packages/tenacity-git/tenacity-git.pacscript [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+        sudo make -j$(nproc)
+}
+
+install() {
+          sudo make install DESTDIR="$STOWDIR/$name"
+}
+
+postinst(){
+          true
+}
```
Comment (wizard-28) [inline packages/tenacity-git/tenacity-git.pacscript:35]:
```suggestion
```

An empty `postinst()` is not required.

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/tenacity-git/tenacity-git.pacscript [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+          sudo pip3 install conan
+}
+
+build() {
+        sudo mkdir build
+        cd build
+        sudo cmake -G "Unix Makefiles" \
+        -DCMAKE_BUILD_TYPE=Release \
+        -Duse_ffmpeg=loaded .. 
+        sudo make -j$(nproc)
```
Comment (wizard-28) [inline packages/tenacity-git/tenacity-git.pacscript:26]: Can you try specifying `DESTDIR` in `cmake` and `make` here too?

Comment (wizard-28) [inline packages/tenacity-git/tenacity-git.pacscript:26]:
```suggestion
        cmake -G "Unix Makefiles" \
        -DCMAKE_BUILD_TYPE=Release \
        -Duse_ffmpeg=loaded .. 
        make -j$(nproc)
```

Building with `sudo` will break things, remove it.

File: packages/tenacity-git/tenacity-git.pacscript [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+    git ls-remote "${url}" master | cut -f1 
+}
+version="$(pkgver)"
+
+prepare() {
+          sudo pip3 install conan
+}
+
+build() {
+        sudo mkdir build
```
Comment (wizard-28) [inline packages/tenacity-git/tenacity-git.pacscript:21]:
```suggestion
         mkdir build
```
`sudo` not required.

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Review by wizard-28 — COMMENTED** [review]
Why is this PR also trying to make changes to `make-a-pacscript.md`. Your #77 PR already does that.

File: packages/tenacity-git/tenacity-git.pacscript [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+    git ls-remote "${url}" master | cut -f1 
+}
+version="$(pkgver)"
+
+prepare() {
+          sudo pip3 install conan
+}
+
+build() {
+        mkdir build
```
Comment (D-Brox) [inline packages/tenacity-git/tenacity-git.pacscript:21]: Missing `sudo`

**Review by D-Brox — CHANGES_REQUESTED** [review]

File: packages/tenacity-git/tenacity-git.pacscript (hunk 7 above)
Comment (rain1598) [reply packages/tenacity-git/tenacity-git.pacscript:21]: [>`sudo` not required.
-Wizard28](https://github.com/pacstall/pacstall-programs/pull/75#discussion_r675960568)

Comment (D-Brox) [reply packages/tenacity-git/tenacity-git.pacscript:21]: On develop branch for 1.6, it's not required, but on master branch, 1.5., it still needs it

Comment (D-Brox) [reply packages/tenacity-git/tenacity-git.pacscript:21]: At least for the CI to work

Comment (D-Brox) [reply packages/tenacity-git/tenacity-git.pacscript:21]: If you plan to merge this one in the near future, add it, if not, keep it the way it is

File: packages/tenacity-git/tenacity-git.pacscript [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+version="$(pkgver)"
+
+prepare() {
+    sudo pip3 install conan
+}
+
+build() {
+    sudo mkdir build
+    cd build
+    sudo cmake -G "Unix Makefiles" \
```
Comment (wizard-28) [inline packages/tenacity-git/tenacity-git.pacscript:23]: The script is fine but is `sudo` finally required?

When I run scripts like opendoas-git on my machine it works as intended, but when you check the CI logs it says permission denied etc. So what's going on?

Comment (rain1598) [reply packages/tenacity-git/tenacity-git.pacscript:23]: the CI is on the master branch, the logname fix is still on the develop branch

Comment (rain1598) [reply packages/tenacity-git/tenacity-git.pacscript:23]: > If you plan to merge this one in the near future, add it.
If not, keep it the way it is currently

**Comment by Henryws** [comment]
Is this ready to merge?

**Comment by rain1598** [comment]
yes for 1.5.1, no for 1.6

**Comment by Henryws** [comment]
> yes for 1.5.1, no for 1.6

what do you mean?

**Comment by Henryws** [comment]
update the packagelist

**Comment by rain1598** [comment]
*the last commit was to update the packagelist I forgot to write a commit message

**Pull request merged**